crit/utils_linux.go

```diff
@@ -0,0 +1,34 @@
+//go:build linux
+
+package crit
+
+import "syscall"
```

Comment:
The `syscall` package has been superseded by the `x/sys/unix` package, AFAIK. The same constants are available, we should be using the latter.